src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -12529,7 +12529,7 @@ void func_8084E3C4(Player* this, PlayState* play) {
         if (this->stateFlags2 & (PLAYER_STATE2_23 | PLAYER_STATE2_25)) {
             this->stateFlags2 |= PLAYER_STATE2_24;
         } else {
-            func_8010BD58(play, OCARINA_ACTION_FREE_PLAY);
+            Message_StartOcarina(play, OCARINA_ACTION_FREE_PLAY);
         }
         return;
```

Comment:
Just commenting to avoid duplicate work that I'm interested in documenting this stateFlags and all the ocarina code from player (See my WIP here):

https://github.com/zeldaret/oot/compare/master...engineer124:ocarina?expand=1